src/style.css

```diff
@@ -0,0 +1,18 @@
+* {
+  margin: 0;
+}
+
+.block-1 {
+  height: 100px;
+  background-color: red;
+}
+
+.block-2 {
```

Comment:
`block-2` bad name for a class

Reply:
Thank yo for review, i corrected mistakes